### DISCUSSION TITLE
LoraWAN integration

### DISCUSSION THREAD

**Discussion by endlessuniverse** [crossbario/crossbar#1979] [Ideas]
Would it be possible to bridge Crossbar with a LoraWAN gateway?
Thanks a lot!

**Comment by oberstet** [comment]
Yes, that would be possible using IPv6/UDP over LoRaWAN - this is done with SCHC https://datatracker.ietf.org/doc/html/draft-ietf-lpwan-schc-over-lorawan

The main thing to make that work is a UDP based WAMP transport. Currently we only have bytestream based transports like TCP, TLS or Serial, combined with WebSocket or RawSocket message framing.
